.github/workflows/Shelton17/count_zeros.py

```diff
@@ -0,0 +1,8 @@
+import numpy as np
```

Comment:
Though the expectation here was that you write your own function and not use imported ones.

Reply:
I could modify it and not use an inbuilt function. Is that required to get into the GoL clinic because I am yet to receive any meeting or cohort details?